flare/struc.py

```diff
@@ -433,7 +473,7 @@ def from_file(file_name, format='') -> Union['flare.struc.Structure',
         try:
             with open(file_name, 'r') as _:
                 pass
-        except:
+        except FileNotFoundError:
```

Comment:
Nice catch! Literally!

Reply:
This try/except block may be redundant, since I think the same result could be achieved with:
`with open(file_name, 'r') as _:
    pass`

Reply:
Lol. You're absolutely right.